homework/lesson 6/src/ru/otus/java/homework/HomeworkTemplate.java

```diff
@@ -0,0 +1,56 @@
+package ru.otus.java.homework;
+
+import java.util.Scanner;
+
+/**
+ * Заготовка для выполнения ДЗ "Система тестирования".
+ */
+public class HomeworkTemplate {
+    public static void main(String[] args) {
+        // Переменные для хранения количества правильных и неправильных ответов
+        int correctCount = 0, wrongCount = 0;
+        final String correct = "Правильно";
+        final String incorrect = "Неправильно";
+        final String err_answer = "Такого варианта ответа не существует";
+
+        String[] questions = {"сколько дней в неделе?", "можно ли пройти ревью с первого раза?"};
+
+        String[][]answerOptions = {{"6","7","8","9"},{"да", "нет"}};
+
+        int [] correctAnswers = {2, 2};
+
+        Scanner scanner = new Scanner(System.in);
+        String v_answer;
```

Comment:
В Java не принято разделять слова в переменной (не константе) подчеркиванием. Принято использовать `camelCase`
```suggestion
        String vAnswer;
```

Но здесь и название переменной странное и непонятное: что такое "v"? 
И лучше эту переменную объявить внутри цикла, где она и используется.
Чем уже область видимости - тем лучше, т.к. меньше вероятность совершить ошибку.

Reply:
извиняюсь
это у нас так в oracle принято обозначать локальные переменные
профдеформация